frontend/src/lib/components/SelectDownIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { DownOutlined } from '@ant-design/icons'
+
+export const SelectDownIcon = (props: React.HTMLAttributes<HTMLSpanElement>): JSX.Element => {
```

Comment:
Think it would be useful to clarify this component (i.e. add a short description) for future reference.